android/filament-android/src/main/java/com/google/android/filament/Texture.java

```diff
@@ -341,6 +341,12 @@ public Builder rgbm(boolean enabled) {
             return this;
         }
 
+        @NonNull
+        public Builder usage(long flags) {
```

Comment:
You should document that the long here must be a combination of Usage.* flags.

Reply:
Also does this need to be a long?

Reply:
I'll change this to `byte` to be consistent with the C++.  I was using `long` because I was looking at our existing SwapChain flags for inspiration.

Reply:
Avoid byte like the plague in Java, they are signed and expand to int as soon as you doing bitwise operations on them. You should go with `int` fo rnow.

Reply:
Yeah, I found out the hard way that kotlin won't let me do `or` with byte.

Reply:
Done and done.